api/db/migrations/20251110104207_create_table_sco-blocked-access-dates-table.js

```diff
@@ -0,0 +1,25 @@
+const TABLE_NAME = 'sco-blocked-access-dates';
```

Comment:
```suggestion
const TABLE_NAME = 'sco_blocked_access_dates';
```

C'est le formatage qu'on a utilisé pour la table `certification_versions`.
@Steph0 pour toi c'est la norme à respecter dorénavant ?

Reply:
c'est ce qui est documenté ici https://github.com/1024pix/pix/blob/dev/docs/database.md avec 

> 
> Bon :
> ```
> client_applications
> lti_platform_registrations
> ```
> Pas bon :
> ```
> wonder-assets
> wonder_asset
> wonderAssets
> ```

Reply:
Yep underscores pour les nouvelles tables ! :)

Reply:
Je l'ai changé